src/OceanSimulations/OceanSimulations.jl

```diff
@@ -44,7 +45,7 @@ default_free_surface(grid) = SplitExplicitFreeSurface(grid; cfl=0.7)
 
 # 70 substeps is a safe rule of thumb for an ocean at 1/4 - 1/10th of a degree
 # TODO: pass the cfl and a given Δt to calculate the number of substeps?
-default_free_surface(grid::TripolarGrid) = SplitExplicitFreeSurface(grid; substeps = 70)
+default_free_surface(grid::Union{TripolarGrid, DistributedGrid}) = SplitExplicitFreeSurface(grid; substeps = 70)
```

Comment:
Why not just make this always the default?

Note that this doesn't cover ImmersedBoundaryGrid but it needs to (which is done on #230)

All these changes need tests I think.